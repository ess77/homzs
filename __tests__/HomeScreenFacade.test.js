import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import HomeScreenFacade from '../components/HomeScreenFacade2';

test('Home Screen Test', () => {
    const snap = renderer.create(
        <HomeScreenFacade />
    ).toJSON();

    expect(snap).toMatchSnapshot();
});
