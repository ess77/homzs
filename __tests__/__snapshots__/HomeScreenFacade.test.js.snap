// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Screen Test 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    accessibilityIgnoresInvertColors={true}
    style={
      Object {
        "flex": 1,
        "height": "100%",
        "width": "100%",
      }
    }
  >
    <Image
      source={1}
      style={
        Array [
          Object {
            "bottom": 0,
            "left": 0,
            "position": "absolute",
            "right": 0,
            "top": 0,
          },
          Object {
            "height": "100%",
            "width": "100%",
          },
          undefined,
        ]
      }
    />
    <View
      style={
        Object {
          "backgroundColor": "rgba(231,223,216,0.5)",
          "flex": 1,
        }
      }
    >
      <View
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
            "justifyContent": "space-around",
            "marginTop": 125,
            "opacity": 0.8,
          }
        }
      >
        <Image
          source={1}
          style={
            Object {
              "borderRadius": 100,
              "height": 200,
              "width": 200,
            }
          }
        />
        <Text
          style={
            Object {
              "color": "white",
              "fontSize": 32,
              "marginTop": -20,
            }
          }
        >
          TinderHouse
        </Text>
        <Text
          style={
            Object {
              "fontSize": 15,
              "marginTop": 30,
            }
          }
        >
          Vente Rapide  -  Achat Rapide
        </Text>
      </View>
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 3,
            "justifyContent": "center",
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "column",
              "justifyContent": "space-around",
            }
          }
        >
          <View
            style={
              Object {
                "color": "rgba(231,223,216,0.5)",
                "marginTop": 50,
              }
            }
          >
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "rgba(70,44,8,.5)",
                  "borderColor": "rgba(70,44,8,.5)",
                  "borderRadius": 30,
                  "borderWidth": 1,
                  "justifyContent": "center",
                  "marginHorizontal": 20,
                  "marginVertical": 10,
                  "opacity": 1,
                  "paddingVertical": 10,
                  "width": 200,
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#fff",
                      "fontSize": 25,
                      "fontWeight": "400",
                    },
                    Object {
                      "color": "#fff",
                      "fontSize": 25,
                      "fontWeight": "500",
                    },
                    Object {
                      "color": "#fff",
                      "fontSize": 25,
                      "fontWeight": "400",
                    },
                  ]
                }
              >
                Inscription
              </Text>
            </View>
          </View>
          <View
            style={
              Object {
                "color": "rgba(231,223,216,0.5)",
                "marginTop": 15,
              }
            }
          >
            <View
              accessible={true}
              focusable={true}
              onClick={[Function]}
              onResponderGrant={[Function]}
              onResponderMove={[Function]}
              onResponderRelease={[Function]}
              onResponderTerminate={[Function]}
              onResponderTerminationRequest={[Function]}
              onStartShouldSetResponder={[Function]}
              style={
                Object {
                  "alignItems": "center",
                  "backgroundColor": "rgba(70,44,8,.5)",
                  "borderColor": "rgba(70,44,8,.5)",
                  "borderRadius": 30,
                  "borderWidth": 1,
                  "justifyContent": "center",
                  "marginHorizontal": 20,
                  "marginVertical": 10,
                  "opacity": 1,
                  "paddingVertical": 5,
                  "width": 200,
                }
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "color": "#fff",
                      "fontSize": 25,
                      "fontWeight": "400",
                    },
                    Object {
                      "color": "#fff",
                      "fontSize": 25,
                      "fontWeight": "500",
                    },
                    Object {
                      "color": "#fff",
                      "fontSize": 12,
                      "fontWeight": "500",
                    },
                  ]
                }
              >
                Connexion
              </Text>
            </View>
          </View>
        </View>
      </View>
      <View
        style={
          Object {
            "marginTop": 35,
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "flexDirection": "row",
              "justifyContent": "space-around",
              "width": "100%",
            }
          }
        >
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "rgba(70,44,8,.5)",
                "borderColor": "rgba(70,44,8,.5)",
                "borderRadius": 30,
                "borderWidth": 1,
                "justifyContent": "center",
                "marginHorizontal": 20,
                "marginVertical": 10,
                "opacity": 1,
                "paddingVertical": 5,
                "width": 125,
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#fff",
                    "fontSize": 25,
                    "fontWeight": "400",
                  },
                  Object {
                    "color": "#fff",
                    "fontSize": 25,
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#fff",
                    "fontSize": 12,
                    "fontWeight": "500",
                  },
                ]
              }
            >
              Recherche
            </Text>
          </View>
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "rgba(70,44,8,.5)",
                "borderColor": "rgba(70,44,8,.5)",
                "borderRadius": 30,
                "borderWidth": 1,
                "justifyContent": "center",
                "marginHorizontal": 20,
                "marginVertical": 10,
                "opacity": 1,
                "paddingVertical": 5,
                "width": 125,
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#fff",
                    "fontSize": 25,
                    "fontWeight": "400",
                  },
                  Object {
                    "color": "#fff",
                    "fontSize": 25,
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#fff",
                    "fontSize": 12,
                    "fontWeight": "500",
                  },
                ]
              }
            >
              Selection
            </Text>
          </View>
          <View
            accessible={true}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "alignItems": "center",
                "backgroundColor": "rgba(70,44,8,.5)",
                "borderColor": "rgba(70,44,8,.5)",
                "borderRadius": 30,
                "borderWidth": 1,
                "justifyContent": "center",
                "marginHorizontal": 20,
                "marginVertical": 10,
                "opacity": 1,
                "paddingVertical": 5,
                "width": 125,
              }
            }
          >
            <Text
              style={
                Array [
                  Object {
                    "color": "#fff",
                    "fontSize": 25,
                    "fontWeight": "400",
                  },
                  Object {
                    "color": "#fff",
                    "fontSize": 25,
                    "fontWeight": "500",
                  },
                  Object {
                    "color": "#fff",
                    "fontSize": 12,
                    "fontWeight": "500",
                  },
                ]
              }
            >
              Transactions
            </Text>
          </View>
        </View>
      </View>
      <View>
        <Text
          style={
            Object {
              "fontWeight": "bold",
              "textAlign": "center",
            }
          }
        >
          Copyright TinderHouzze© 2020
        </Text>
      </View>
    </View>
  </View>
</View>
`;
